posthog/caching/warming.py

```diff
@@ -82,13 +88,24 @@ def priority_insights(team: Team) -> Generator[tuple[int, Optional[int]], None,
 @shared_task(ignore_result=True, expires=60 * 15)
 def schedule_warming_for_teams_task():
     team_ids = largest_teams(limit=10)
+    threshold = datetime.now(UTC) - LAST_VIEWED_THRESHOLD
 
-    teams = Team.objects.filter(Q(pk__in=team_ids) | Q(extra_settings__insights_cache_warming=True))
+    prio_teams = Team.objects.filter(Q(pk__in=team_ids) | Q(extra_settings__insights_cache_warming=True))
+    teams_with_recently_viewed_shared = Team.objects.filter(
```

Comment:
For shared insights, we don't prioritise just the `largest_teams`, so this should pull a lot of teams (any which have their shared insights viewed recently), unless I missed something?
Also maybe we can filter for top insights/dashboards which are viewed the most?

Reply:
Yes, we pull in all teams with recently viewed shared insights. I can run the numbers again, but it should be a couple of hundred teams, not sure how many insights. I don't think this is too much, also see my not regarding safeguards, above. For comparison, the old warming system does something like 750 insight updates per hour (the ones that I think are not every effective/targeted). If we switch these off we reduce load again.

Reply:
Understood, thanks for explaining. When compared to the old system not much at all. Can later on filter for most viewed insights/dashboards if needed. Also makes sense with your note about running on one pod.